app/models/application_record.rb

```diff
@@ -8,6 +8,11 @@ class ApplicationRecord < ActiveRecord::Base
     meta_example = ", e.g. #{@meta[:example]}" if @meta[:example]
     name_desc = @meta[:name_desc] || "Name of the #{@meta[:friendly_name] || @meta[:class_scope]}#{meta_example}"
     property :name, String, desc: name_desc
+    property :present?, one_of: [true, false], desc: 'Object presence (always true)'
+  end
+
+  class Jail < Safemode::Jail
+    allow :id, :name, :present?
```

Comment:
I could not find where exactly `id` is allowed, the point of this hunk is that this class has documentation that says these are all allowed (I also added present? on top of that which is safe to add). Problem is, it was unclear where it gets added. So I wanted to explicitly name all three here, Safemode seems to walk up the class hierarchy when finding allowed methods and it is okay to "override" them.

Do you prefer not to mention `name` since it is allowed for all Ruby objects? For the record, we do actually allow it in the individual model classes anyway, so this is not a deviation from our approach. I prefer clean Jail here which matches the documentation rather than partial list to be honest.